improver_tests/acceptance/test_regrid.py

```diff
@@ -203,3 +203,91 @@ def test_regrid_nearest_landmask_multi_realization(tmp_path):
     ]
     run_cli(args)
     acc.compare(output_path, kgo_path)
+
+
+def test_regrid_nearest_2_multi_realization(tmp_path):
+    """Test nearest-2 neighbour regridding"""
+    kgo_dir = acc.kgo_root() / "regrid"
+    kgo_path = kgo_dir / "nearest_2/kgo_multi_realization.nc"
+    input_path = kgo_dir / "landmask/global_cutout_multi_realization.nc"
+    target_path = kgo_dir / "landmask/ukvx_landmask.nc"
+    output_path = tmp_path / "output.nc"
+    args = [
+        input_path,
+        target_path,
+        "--output",
+        output_path,
+        "--regrid-mode",
+        "nearest-2",
+        "--regridded-title",
+        GLOBAL_UK_TITLE,
+    ]
+    run_cli(args)
+    acc.compare(output_path, kgo_path)
```

Comment:
Comparing nearest-neighbour results is tricky. As I mentioned,  there are alternative points with equal distance, and it will possibly lead to different results at some points. In the five Jupyter notebooks I gave, 3/5 are the same, and 2/5 different. I debugged this kind of cases and all pointed to the points with equal distance. 

We chose a multi-realization case for avoiding the comparison, and also for more general cases.